diff --git a/Makefile b/Makefile
index 97f9338..d56fc44 100644
--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ bup: lib/bup/_version.py lib/bup/_helpers$(SOEXT) cmds
 Documentation/all: bup
 
 INSTALL=install
-PYTHON=python
+PYTHON=python2
 PREFIX=/usr
 MANDIR=$(DESTDIR)$(PREFIX)/share/man
 DOCDIR=$(DESTDIR)$(PREFIX)/share/doc/bup
diff --git a/cmd/bloom-cmd.py b/cmd/bloom-cmd.py
index cd4506d..68dab80 100755
--- a/cmd/bloom-cmd.py
+++ b/cmd/bloom-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, glob, tempfile
 from bup import options, git, bloom
 from bup.helpers import *
diff --git a/cmd/cat-file-cmd.py b/cmd/cat-file-cmd.py
index 8948ff2..b1aa78b 100755
--- a/cmd/cat-file-cmd.py
+++ b/cmd/cat-file-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, stat
 from bup import options, git, vfs
 from bup.helpers import *
diff --git a/cmd/daemon-cmd.py b/cmd/daemon-cmd.py
index b92716f..045f7e1 100755
--- a/cmd/daemon-cmd.py
+++ b/cmd/daemon-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, getopt, socket, subprocess, fcntl
 from bup import options, path
 from bup.helpers import *
diff --git a/cmd/damage-cmd.py b/cmd/damage-cmd.py
index 49dbed0..a1e6c2d 100755
--- a/cmd/damage-cmd.py
+++ b/cmd/damage-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, random
 from bup import options
 from bup.helpers import *
diff --git a/cmd/drecurse-cmd.py b/cmd/drecurse-cmd.py
index cdf6680..16af3db 100755
--- a/cmd/drecurse-cmd.py
+++ b/cmd/drecurse-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 from os.path import relpath
 from bup import options, drecurse
diff --git a/cmd/fsck-cmd.py b/cmd/fsck-cmd.py
index 794c755..e68f624 100755
--- a/cmd/fsck-cmd.py
+++ b/cmd/fsck-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, glob, subprocess
 from bup import options, git
 from bup.helpers import *
diff --git a/cmd/ftp-cmd.py b/cmd/ftp-cmd.py
index baa1e08..846649d 100755
--- a/cmd/ftp-cmd.py
+++ b/cmd/ftp-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, stat, fnmatch
 from bup import options, git, shquote, vfs, ls
 from bup.helpers import *
diff --git a/cmd/fuse-cmd.py b/cmd/fuse-cmd.py
index 30776cb..b0ec725 100755
--- a/cmd/fuse-cmd.py
+++ b/cmd/fuse-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, errno
 from bup import options, git, vfs, xstat
 from bup.helpers import *
diff --git a/cmd/help-cmd.py b/cmd/help-cmd.py
index 56a4148..ed7cd61 100755
--- a/cmd/help-cmd.py
+++ b/cmd/help-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, glob
 from bup import options, path
 
diff --git a/cmd/index-cmd.py b/cmd/index-cmd.py
index b7fe8f1..4924c06 100755
--- a/cmd/index-cmd.py
+++ b/cmd/index-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import sys, stat, time, os, errno, re
 from bup import metadata, options, git, index, drecurse, hlinkdb
diff --git a/cmd/init-cmd.py b/cmd/init-cmd.py
index 2e4a151..cd86612 100755
--- a/cmd/init-cmd.py
+++ b/cmd/init-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys
 
 from bup import git, options, client
diff --git a/cmd/join-cmd.py b/cmd/join-cmd.py
index 042b402..dfbf5a5 100755
--- a/cmd/join-cmd.py
+++ b/cmd/join-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys
 from bup import git, options, client
 from bup.helpers import *
diff --git a/cmd/list-idx-cmd.py b/cmd/list-idx-cmd.py
index a3e3dc9..a9f6477 100755
--- a/cmd/list-idx-cmd.py
+++ b/cmd/list-idx-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os
 from bup import git, options
 from bup.helpers import *
diff --git a/cmd/ls-cmd.py b/cmd/ls-cmd.py
index 94a7131..b00c41d 100755
--- a/cmd/ls-cmd.py
+++ b/cmd/ls-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys
 from bup import git, vfs, ls
 from bup.helpers import *
diff --git a/cmd/margin-cmd.py b/cmd/margin-cmd.py
index 9b7fd60..798512b 100755
--- a/cmd/margin-cmd.py
+++ b/cmd/margin-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, struct, math
 from bup import options, git, _helpers
 from bup.helpers import *
diff --git a/cmd/memtest-cmd.py b/cmd/memtest-cmd.py
index 0e3cf0c..6798d88 100755
--- a/cmd/memtest-cmd.py
+++ b/cmd/memtest-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, re, struct, time, resource
 from bup import git, bloom, midx, options, _helpers
 from bup.helpers import *
diff --git a/cmd/meta-cmd.py b/cmd/meta-cmd.py
index cac26b5..9d43b8a 100755
--- a/cmd/meta-cmd.py
+++ b/cmd/meta-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Copyright (C) 2010 Rob Browning
 #
diff --git a/cmd/midx-cmd.py b/cmd/midx-cmd.py
index 5c0c126..4874ed7 100755
--- a/cmd/midx-cmd.py
+++ b/cmd/midx-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, math, struct, glob, resource
 import tempfile
 from bup import options, git, midx, _helpers, xstat
diff --git a/cmd/mux-cmd.py b/cmd/mux-cmd.py
index 299dec9..ee82ff9 100755
--- a/cmd/mux-cmd.py
+++ b/cmd/mux-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import os, sys, subprocess, struct
 from bup import options
 from bup.helpers import *
diff --git a/cmd/newliner-cmd.py b/cmd/newliner-cmd.py
index 68d327b..c485ee5 100755
--- a/cmd/newliner-cmd.py
+++ b/cmd/newliner-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, re
 from bup import options
 from bup import _helpers   # fixes up sys.argv on import
diff --git a/cmd/on--server-cmd.py b/cmd/on--server-cmd.py
index 327c81f..bca9ac9 100755
--- a/cmd/on--server-cmd.py
+++ b/cmd/on--server-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, struct
 from bup import options, helpers
 
diff --git a/cmd/on-cmd.py b/cmd/on-cmd.py
index cebff1a..ff9564d 100755
--- a/cmd/on-cmd.py
+++ b/cmd/on-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, struct, getopt, subprocess, signal
 from bup import options, ssh, path
 from bup.helpers import *
diff --git a/cmd/random-cmd.py b/cmd/random-cmd.py
index 4be3660..3fd8a6c 100755
--- a/cmd/random-cmd.py
+++ b/cmd/random-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys
 from bup import options, _helpers
 from bup.helpers import *
diff --git a/cmd/restore-cmd.py b/cmd/restore-cmd.py
index 535d83c..a9d7d43 100755
--- a/cmd/restore-cmd.py
+++ b/cmd/restore-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import copy, errno, sys, stat, re
 from bup import options, git, metadata, vfs
 from bup.helpers import *
diff --git a/cmd/save-cmd.py b/cmd/save-cmd.py
index 8067fac..56e29af 100755
--- a/cmd/save-cmd.py
+++ b/cmd/save-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, stat, time, math
 from cStringIO import StringIO
 from errno import EACCES
diff --git a/cmd/server-cmd.py b/cmd/server-cmd.py
index 43a3fce..10ec142 100755
--- a/cmd/server-cmd.py
+++ b/cmd/server-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import os, sys, struct
 from bup import options, git
 from bup.helpers import *
diff --git a/cmd/split-cmd.py b/cmd/split-cmd.py
index 70daf7c..a85428e 100755
--- a/cmd/split-cmd.py
+++ b/cmd/split-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import os, sys, time
 from bup import hashsplit, git, options, client
 from bup.helpers import *
diff --git a/cmd/tag-cmd.py b/cmd/tag-cmd.py
index 28fa328..5e55829 100755
--- a/cmd/tag-cmd.py
+++ b/cmd/tag-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 """Tag a commit in the bup repository.
 Creating a tag on a commit can be used for avoiding automatic cleanup from
 removing this commit due to old age.
diff --git a/cmd/tick-cmd.py b/cmd/tick-cmd.py
index 4d462ad..7b24d1f 100755
--- a/cmd/tick-cmd.py
+++ b/cmd/tick-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, time
 from bup import options
 
diff --git a/cmd/version-cmd.py b/cmd/version-cmd.py
index 645f25f..fcc2b95 100755
--- a/cmd/version-cmd.py
+++ b/cmd/version-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys
 from bup import options
 from bup import _version
diff --git a/cmd/web-cmd.py b/cmd/web-cmd.py
index c94b7c6..13a4512 100755
--- a/cmd/web-cmd.py
+++ b/cmd/web-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, stat, urllib, mimetypes, posixpath, time, webbrowser
 from bup import options, git, vfs
 from bup.helpers import *
diff --git a/cmd/xstat-cmd.py b/cmd/xstat-cmd.py
index 85071d2..2c16d48 100755
--- a/cmd/xstat-cmd.py
+++ b/cmd/xstat-cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (C) 2010 Rob Browning
 #
 # This code is covered under the terms of the GNU Library General
diff --git a/main.py b/main.py
index 45a0e8a..5c7c9d7 100755
--- a/main.py
+++ b/main.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import sys, os, subprocess, signal, getopt
 
 argv = sys.argv
diff --git a/t/cleanup-mounts-under b/t/cleanup-mounts-under
index a8ba611..8af0200 100755
--- a/t/cleanup-mounts-under
+++ b/t/cleanup-mounts-under
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import os.path, re, subprocess, sys
 
diff --git a/t/hardlink-sets b/t/hardlink-sets
index 653b4d2..d3852be 100755
--- a/t/hardlink-sets
+++ b/t/hardlink-sets
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import os, stat, sys
 
diff --git a/t/id-other-than b/t/id-other-than
index cfa219d..cb76717 100755
--- a/t/id-other-than
+++ b/t/id-other-than
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import grp
 import pwd
diff --git a/t/mksock b/t/mksock
index ee2dbfe..370b753 100755
--- a/t/mksock
+++ b/t/mksock
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import socket, sys
 
diff --git a/t/ns-timestamp-resolutions b/t/ns-timestamp-resolutions
index d1bb785..2b40243 100755
--- a/t/ns-timestamp-resolutions
+++ b/t/ns-timestamp-resolutions
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import os, sys
 
diff --git a/t/root-status b/t/root-status
index 9fb12cb..715a651 100755
--- a/t/root-status
+++ b/t/root-status
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import sys
 
diff --git a/t/unknown-owner b/t/unknown-owner
index 212d146..90c177b 100755
--- a/t/unknown-owner
+++ b/t/unknown-owner
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import grp
 import pwd
diff --git a/wvtest.py b/wvtest.py
index cc68601..3f799ba 100755
--- a/wvtest.py
+++ b/wvtest.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # WvTest:
 #   Copyright (C)2007-2012 Versabanq Innovations Inc. and contributors.
